git/objects/commit.py

```diff
@@ -335,8 +336,70 @@ def stats(self) -> Stats:
         return Stats._list_from_string(self.repo, text)
 
     @property
-    def trailers(self) -> Dict:
-        """Get the trailers of the message as dictionary
+    def trailers(self) -> Dict[str, str]:
+        """Get the trailers of the message as a dictionary
+
+        Git messages can contain trailer information that are similar to RFC 822
+        e-mail headers (see: https://git-scm.com/docs/git-interpret-trailers).
+
+        WARNING: This function only returns the latest instance of each trailer key
+        and will be deprecated soon. Please see either ``Commit.trailers_list`` or ``Commit.trailers_dict``.
+
+        :return:
+            Dictionary containing whitespace stripped trailer information.
+            Only the latest instance of each trailer key.
+        """
+        return {
+            k: v[0] for k, v in self.trailers_dict.items()
+        }
+
+    @property
+    def trailers_list(self) -> List[str]:
+        """Get the trailers of the message as a list
+
+        Git messages can contain trailer information that are similar to RFC 822
+        e-mail headers (see: https://git-scm.com/docs/git-interpret-trailers).
+
+        This functions calls ``git interpret-trailers --parse`` onto the message
+        to extract the trailer information, returns the raw trailer data as a list.
+
+        Valid message with trailer::
+
+            Subject line
+
+            some body information
+
+            another information
+
+            key1: value1.1
+            key1: value1.2
+            key2 :    value 2 with inner spaces
+
+
+        Returned list will look like this::
+
+            [
+                "key1: value1.1",
+                "key1: value1.2",
+                "key2 :    value 2 with inner spaces",
+            ]
+
+
+        :return:
+            List containing whitespace stripped trailer information.
+        """
+        cmd = ["git", "interpret-trailers", "--parse"]
+        proc: Git.AutoInterrupt = self.repo.git.execute(cmd, as_process=True, istream=PIPE)  # type: ignore
+        trailer: str = proc.communicate(str(self.message).encode())[0].decode()
```

Comment:
`.decode()` will probably use the interpreter encoding which I hope is UTF8 by now for the greatest changes of success. If not, I think the expected encoding should be specified. It's worth noting that `git` doesn't enforce any encoding, even though commits may have encoding hints embedded as header field. It's a bit of a mess to deal with this correctly here, so a best-guess is all we can reasonably do here. (This is not `gitoxide`, unfortunately).

Reply:
Wasn't sure exactly what you were asking for here 😅 but I've specified a decoding - [ed36bd9](https://github.com/gitpython-developers/GitPython/pull/1576/commits/ed36bd903e1fdf45c53b52dbcb1b2d8444965d98)